android/src/main/res/layout/subtitles.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+  ~ Copyright (c) 2019 Magic Leap, Inc. All Rights Reserved
```

Comment:
Should be used new licence: `2019-2020` according to [this PR](https://github.com/magic-script/magic-script-components-react-native/pull/503)